### PR TITLE
migrate edm2me converter to consumes

### PR DESCRIPTION
**Pull request by deguio**
I know it doesn't look very good, but I didn't have a smarter idea. adding a bunch of experts:
@Dr15Jones @davidlange6 @rovere @VinInn

### CODE REVIEW

**Comment by VinInn** [comment]
template and specialized instead of runtime?

File: DQMServices/Components/plugins/EDMtoMEConverter.cc [hunk 1]
```diff
@@ -152,7 +184,10 @@ EDMtoMEConverter::getData(T& iGetFrom, bool iEndRun)
 
     if (classtypes[ii] == "TH1F") {
       edm::Handle<MEtoEDM<TH1F> > metoedm;
-      iGetFrom.getByLabel(*inputTag, metoedm);
+      if(iEndRun)
+	iGetFrom.getByToken(runInputTagTH1F_, metoedm);
```
Comment (VinInn) [inline DQMServices/Components/plugins/EDMtoMEConverter.cc]: iGetFrom.getByToken(iEndRun ? runInputTagTH1F_ : lumiInputTagTH1F_, metoedm)

**Comment by deguio** [comment]
ciao @VinInn 
not sure I understand your first comment. you mean to make the EDMtoMEConverter class template?
thanks,
F.

**Comment by makortel** [comment]
Here is my take for https://github.com/cms-sw/cmssw/pull/10307#issuecomment-123775409 (couldn't resist to try) https://github.com/makortel/cmssw/commit/6245aa6 (probably at the extreme). It compiles on gcc and clang, but I didn't test running.

**Comment by VinInn** [comment]
Very close to what I add in mind...

**Comment by deguio** [comment]
thanks guys, for the precious comments.
I will update the code accordingly

F.

**Pull request closed without merging**